kubernetes/process_manifest.rb

```diff
@@ -3,6 +3,7 @@
 require "util"
 require "yaml"
 require "fileutils"
+require 'json'
```

Comment:
Oh, I learned to do it for security reasons, not performance reasons :) (potential information disclosure concerns)

Are these conventions documented somewhere? It would help to be able to read them :)